Bluepill-cli/bluepill-cli-release.xcconfig

```diff
@@ -1,10 +0,0 @@
-//  Copyright 2016 LinkedIn Corporation
-//  Licensed under the BSD 2-Clause License (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at https://opensource.org/licenses/BSD-2-Clause
-//
-//  Unless required by applicable law or agreed to in writing, software
-//  distributed under the License is distributed on an "AS IS" BASIS,
-//  WITHOUT WARRANTIES OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations under the License.
-
-#include "bluepill-cli.xcconfig"
```

Comment:
where are the xcconfig now?

Reply:
They are in each directory, I just renamed it to make it obvious which project they belong to:

```
$ find . -name \*.xcconfig
./Configurations/common.xcconfig
./bp/bp.xcconfig
./bluepill/bluepill.xcconfig
```